src/screens/Assets/collectibles/CollectiblesTab.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { SectionList, useWindowDimensions } from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import styled from 'styled-components/native';
+import { chunk } from 'lodash';
+import { useTranslation } from 'translations/translate';
+
+// Components
+import FloatingButtons from 'components/FloatingButtons';
+import Modal from 'components/Modal';
+import ReceiveModal from 'screens/Asset/ReceiveModal';
+
+// Constants
+import { COLLECTIBLE, SEND_COLLECTIBLE_FROM_ASSET_FLOW } from 'constants/navigationConstants';
+
+// Selectors
+import { useRootSelector, activeAccountAddressSelector } from 'selectors';
+import { useSupportedChains } from 'selectors/smartWallet';
+
+// Utils
+import { spacing } from 'utils/variables';
+
+// Types
+import type { SectionBase } from 'utils/types/react-native';
+import type { Chain } from 'models/Chain';
+
+// Local
+import { type FlagPerChain, useExpandItemsPerChain } from '../utils';
+import ChainListHeader from '../components/ChainListHeader';
+import ChainListFooter from '../components/ChainListFooter';
+import { type CollectibleItem, useCollectibleAssets } from './selectors';
+import { buildCollectibleNavigationParams } from './utils';
+import CollectibleListItem from './CollectibleListItem';
+
+function CollectiblesTab() {
+  const { t } = useTranslation();
+  const navigation = useNavigation();
+  const safeArea = useSafeAreaInsets();
+
+  const { chain: initialChain } = navigation.state.params;
```

Comment:
I wonder since we started using `useNavigation` hook maybe we can start using `useNavigationParam` or `useNavigationState` instead of destructuring `navigation`?

Reply:
I've used navigation.getParam() to slightly improve type checking.